vcf.cc

```diff
@@ -64,15 +64,14 @@ vcf_print_utf(const utf_string* s)
     return;
   }
 
-  char* stripped_html = strip_html(s);
-  char* p = gstrsub(stripped_html, "\n", "\\n");
-  char* p2 = gstrsub(p, "<p>", "\\n");
-  char* p3 = gstrsub(p2, ";", "\\;");
-  gbfputs(p3, file_out);
-  xfree(p);
-  xfree(p2);
-  xfree(p3);
-  xfree(stripped_html);
+  char *string = strip_html(s);
+  QString stripped_html = string;
+  xfree(string);
+
+  stripped_html.replace("\n", "\\n");
+  stripped_html.replace("<p>", "\\n");
+  stripped_html.replace(";", "\\;");
```

Comment:
I think commas need to be escaped as well, see https://datatracker.ietf.org/doc/html/rfc2426#section-2.4.2
`stripped_html.replace(",", "\\,");`